src/kernels/raw/session/rawSocket.node.ts

```diff
@@ -100,7 +101,7 @@ export class RawSocket implements IWebSocketLike, IKernelSocket, IDisposable {
         // If from ipywidgets, this will be serialized already, so turn it back into a message so
         // we can add the special hash to it.
         const message = this.deserialize(data);
-
+        this.onAnyMessage(message as any);
```

Comment:
When we get messages from the UI kenel, we send that directly to the kernel bypassing the Jupyter lab API
As a result we don't invoke the event `anymessage` (which happens in jupyter code when messages are sent to the kernel)

For https://github.com/microsoft/vscode-jupyter/issues/9503, we need the ability to monitor all messagesa & anymessage event handler will pick up some of these messages which only works in Jupyter & not in raw kernels.

Basically this fixes an issue.

Reply:
Can you put this statement as a comment? I'm probably going to wonder why we're calling this.onAnyMessage here later on.